packages/explorer/src/app/(explorer)/[chainName]/worlds/[worldAddress]/explore/utils/decodeTable.ts

```diff
@@ -0,0 +1,45 @@
+import { Hex, decodeAbiParameters, parseAbiParameters } from "viem";
+import { hexToResource } from "@latticexyz/common";
+import { Schema, Table } from "@latticexyz/config";
+import { hexToSchema } from "@latticexyz/protocol-parser/internal";
+
+export type DeployedTable = Omit<Table, "label" | "namespaceLabel"> & {
+  valueSchema: Schema;
+  keySchema: Schema;
+};
+
+export const decodeTable = (row: Hex[]): DeployedTable => {
```

Comment:
what's this for? sqlite specifically or dozer specifically?

a lot of this logic looks duplicative of https://github.com/latticexyz/mud/blob/main/packages/store-sync/src/logToTable.ts

no change needed here, can do this as a follow up, but thinking we could lift this out into protocol-parser or something so you can go from the onchain representation of a table (a record from `Tables` table) into a proper table object

Reply:
This was originally used for Dozer endpoint since it returns all the data in hex form. It's now used for SQLite as well due to a different approach of how it's being fetched i.e. the data for the table is now coming from `store__Tables`.